### PR TITLE
[master][KOGITO-1673] - Images should deploy version without micro version in tag

### PR DESCRIPTION
**Pull request by spolti**
see: https://issues.redhat.com/browse/KOGITO-1673
new tag already available on quay for all images.

Signed-off-by: spolti <fspolti@redhat.com>

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 IMAGE_VERSION := $(shell cat image.yaml | egrep ^version  | cut -d"\"" -f2)
+SHORTENED_LATEST_VERSION := $(shell echo $(IMAGE_VERSION) | awk -F. '{print $$1"."$$2"-latest"}')
```
Comment (radtriste) [inline Makefile]: [operator](https://github.com/kiegroup/kogito-cloud-operator/pull/260) is setting `M.m.x-latest` which is different from `M.m-latest` that we have here...
@ricardozanini @spolti can we decide on the naming ?

I like the `M.m-latest` to be fair

**Review by radtriste — APPROVED** [review]
apart from the naming convention comment, the rest is ok

File: Makefile (hunk 1 above)
Comment (sutaakar) [reply Makefile]: What about using just `M.m`?

Comment (radtriste) [reply Makefile]: IMHO with the `latest`, we make sure people know we put the latest bug fix there and we update it.
`M.m` could be confusing with `M.m.0`

Comment (sutaakar) [reply Makefile]: hmm, ok, I thought that the tag usually mark latest image in that major/minor/patch version, like in https://hub.docker.com/_/mysql

Comment (spolti) [reply Makefile]: if we're going to use x.y.z-latest, what's the difference from latest?

Comment (radtriste) [reply Makefile]: operator is using `.x-latest` as suffix (x undefined) and you are using `-latest` here.
@sutaakar gave a good example of just using `M.m` so we could that way.

Comment (radtriste) [reply Makefile]: @spolti lets go for `M.m` (no latest suffix)

Comment (radtriste) [reply Makefile]: JIRA updated

File: tests/features/kogito-quarkus-jvm-ubi8.feature [hunk 2]
```diff
@@ -24,25 +24,25 @@ Feature: Kogito-quarkus-ubi8 feature.
       | variable            | value                     |
       | NATIVE              | false                     |
       | JAVA_OPTIONS        | -Dquarkus.log.level=DEBUG |
-    Then check that page is served
+    Then file /home/kogito/bin/drools-quarkus-example-8.0.0-SNAPSHOT-runner.jar should exist
```
Comment (radtriste) [inline tests/features/kogito-quarkus-jvm-ubi8.feature]: you are solving something else there or ?

**Comment by ricardozanini** [comment]
Please change this PR title to: "Images should deploy version without micro version in tag" like in the JIRA.

**Review by ricardozanini — APPROVED** [review]

File: tests/features/kogito-quarkus-jvm-ubi8.feature (hunk 2 above)
Comment (spolti) [reply tests/features/kogito-quarkus-jvm-ubi8.feature]: trying to address the failing tests reported by ci.
that's funny though, each time it run a different test fail =/

**Comment by ricardozanini** [comment]
All of those tests changes are also part of this tagging/naming thing?

File: Makefile (hunk 1 above)
Comment (spolti) [reply Makefile]: tags updated.

**Comment by spolti** [comment]
just a trying to figure out why tests are failing on CI but not locally =//

@sutaakar @radtriste Do you guys have any idea on how we can troubleshoot this failing tests?
Running over and over again locally none try has failed, but on ci it keeps failing.

**Comment by sutaakar** [comment]
@spolti Sorry, I don't have access to the ci, can someone add me there?

**Comment by spolti** [comment]
CI is not stable yet, merging.

**Comment by radtriste** [comment]
any update there ?
operator PRs are already merged...

@spolti the first changes you made in `Makefile` file, shoud not have any impact on tests normally or ?
If that is the case, I think you should remove the changes on feature files and we should focus on this in another issue/PR...

**Comment by spolti** [comment]
they are not related at all, it was just a try to fix it, bu yeah, I agree, will remove.

**Review by radtriste — APPROVED** [review]

**Pull request merged**